E-Commerce-API/app.js

```diff
@@ -1,8 +1,175 @@
 const express = require("express");
 const app = express();
+var bodyParser = require("body-parser");
 // Your code to run the server should go here
 // Don't hardcode your DB password in the code or upload it to GitHub! Never ever do this ever.
 // Use environment variables instead:
 // https://www.codementor.io/@parthibakumarmurugesan/what-is-env-how-to-set-up-and-run-a-env-file-in-node-1pnyxw9yxj
+app.listener = app.listen(3000, function () {
+  console.log(`Server is listening on port ${3000}`);
+});
```

Comment:
Normally this goes at the bottom as I think this is a blocking call, i.e. the rest of the code below it won't run - it's also a little jarring to see it at the top of the file from a convention standpoint.